# add-maintainer-mode.patch by Daniel T Chen <crimsun@ubuntu.com>
# based on prior work by Loic Minier <lool@dooz.org>
diff -uNr alsa-lib-1.0.22.orig/configure.in alsa-lib-1.0.22/configure.in
--- alsa-lib-1.0.22.orig/configure.in	2009-12-16 15:18:51.000000000 +0000
+++ alsa-lib-1.0.22/configure.in	2009-12-24 06:50:44.000000000 +0000
@@ -22,6 +22,8 @@
 
 AC_PREFIX_DEFAULT(/usr)
 
+AM_MAINTAINER_MODE
+
 dnl Checks for programs.
 
 dnl try to gues cross-compiler if not set
